src/helpers/vpn-check.ts

```diff
@@ -1,8 +1,14 @@
 import { execSync } from "child_process";
 
-const webproxyHost = "websenseproxy.internal.ch";
+const webproxyHost = process.env.CH_PROXY_HOST;
```

Comment:
Are we certain all users have this set?

Reply:
I think that we can say that users ought to have run a recent-ish version of dev-env-setup and this variable goes back a long way.

Reply:
Might be worth documenting how to set this in case people don't have it set